michelson.md

```diff
@@ -1849,7 +1849,7 @@ These operations are used internally for implementation purposes.
 ```
 
 ```symbolic
-  rule <k> #Assume(false) => #Bottom ... </k>
+  rule <k> (#Assume(false) ~> _:K) => #Bottom </k>
```

Comment:
We can do:

```
rule <k> #Assume(BEXP) => . ... </k> ensures BEXP
```

Reply:
I was able to get the main proof to pass again using your method. I'm assuming the other proofs will pass as well. I'm not sure why it appeared to not work yesterday.